pysyncrosim/library.py

```
@@ -613,7 +613,10 @@ def datasheets(self, name=None, summary=True, optional=False, empty=False,
             
             return ds
         
-    def delete(self, project=None, scenario=None, force=False, remove_backup=False, remove_publish=False, remove_custom_folders=False):
+    def delete(self, project=None, scenario=None,
+                pid=None, sid=None, data=None, datasheet=None, ids=None,
```

Ruff (0.14.11)

Unused method argument: `pid`

(ARG002)